include/circt/Dialect/Calyx/CalyxLoweringUtils.h

```diff
@@ -779,6 +779,28 @@ class BuildCallInstance : public calyx::FuncOpPartialLoweringPattern {
   ComponentOp getCallComponent(mlir::func::CallOp callOp) const;
 };
 
+/// Predicate information for the floating point comparisons
```

Comment:
Please update the description in the initial comment to be more declarative rather than "This patch tries to...", e.g.,

`Adds floating points comparison op with the Berkeley HardFloat interface.`

Also, if the issue below it is no longer relevant, I'd remove that too.

Reply:
sure, I'll change the final squash commit message to be more informative as you suggested